lib/l10n/intl_en.arb

```diff
@@ -1,9 +1,9 @@
 {
-  "home_tab_top_anime": "Top Anime",
+  "home_tab_top_anime": "Record Lecture",
```

Comment:
`home_tab_top_anime` is now "Record Lecture"? What happened to the "Top Anime" feature?